frontend/src/components/commentSection/CommentSection.jsx

```diff
@@ -272,21 +280,35 @@ const CommentSection = ({loggedInAuthor, commentsId, commentCount, postAuthorId,
                 }
                 try {
                     console.log("POST AUTHOR ID: ", postAuthorId);
-                    await axios.post(postAuthorId + "/inbox/", newComment, {
-                        headers: {
-                          'Authorization': 'Basic ' + team4Authorization
-                        }
-                      })
-                    .then((response) => {
-                    });
+                    // We cannot get the created comment ID because we cannot get a response BECAUSE the above POST is not allowed from remote nodes
+                    var inboxInfo = {
+                      "type": "comment",
+                      "id": commentsId,
+                      "author": loggedInAuthor,
+                  }
+                  await axios.post(postAuthorId + "/inbox/", inboxInfo, {
+                      headers: {
+                        'Authorization': 'Basic ' + team4Authorization
+                      }
+                    })
+                    // The response from Team 0 has no valuable information
+                    // Only returns "author" and "type"
```

Comment:
THIS IS THE RESPONSE FROM A SUCCESSFUL POST
There is nothing useful coming back?
![image](https://user-images.githubusercontent.com/64627383/161458468-5efbb12e-57dd-4bc8-a06e-72a4ebc2553a.png)